modules/sizing-requirements-for-gitops.adoc

```diff
@@ -28,3 +28,47 @@ Optionally, you can also use the ArgoCD custom resource with the `oc` command to
 ----
 oc edit argocd <name of argo cd> -n namespace
 ----
+
+[id="sizing-requirements-for-argocd-redis_{context}"]
+== Sizing requirements for argocd-redis
+
+During the capacity planning stage for your application in the {gitops-title} Operator, you must ensure that an adequate amount of resources such as memory, CPU, and storage are allocated for the `argocd-redis` pod.
+
+The default memory limit for the `argocd-redis` caching server is `256M`. Memory limit is a value that constitutes a fixed amount of memory that can be allocated across all the processes in a pod.
+
+In some instances, the default memory limit in the `argocd-redis` pod may not be sufficient to manage a large number of resources. Therefore, you must increase the memory limit to initialize the pod. After this value is increased, you must monitor the memory metrics of the Redis pod, and change the memory configuration as the application deployment scales up.
+
+.Example command to view the memory configuration for the Redis pod.
+[source,terminal]
+----
+oc get argocd -n openshift-gitops openshift-gitops -o json | jq '.spec.redis.resources'
+----
+
+.Example Output
+[source,terminal]
+----
+{
+    "limits": { #<1>
+        "cpu": "500m",
+        "memory": "256Mi"
+  },
+  "requests": { #<2>
+    "cpu": "250m",
+    "memory": "128Mi"
+  }
+}
+----
+<1> The highest resource limit threshold allocated to the pod 
+<2> The lowest resource limit threshold allocated to the pod
+
+.Example command to change the memory configuration of the Redis pod.
```

Comment:
I might suggest to provide a little bit more information, as follows:
```suggestion
The following example command changes the memory configuration for a Redis pod.  The highest resource limit threshold is set to 8 GiB and the lowest is set to 256 MiB.
```
WDYT?

Reply:
Fixed.